browser/resources/extensions/brave_overrides/extensions_toolbar.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { RegisterStyleOverride } from '//resources/brave/polymer_overriding.js'
+import { html } from '//resources/polymer/v3_0/polymer/polymer_bundled.min.js'
+
+RegisterStyleOverride('extensions-toolbar', html`<style>
```

Comment:
`RegisterStyleOverride` works well here but curious why it didn't work for `cr-toast`...

Reply:
Oh, I read https://github.com/brave/brave-core/pull/23494#discussion_r1596110912 👍🏼